website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,384 @@
+
+##### Peatland Growth Module
+
+This module gets the data from the variables: ```peatland_growth_parameters```, ```peatland_turnover_parameters``` and ```peatland_growth_curve```. It simulates the various growth cycles, growth curves and also sets the component value for the pools.  For example: woody layer, moss layer and many more.
+
+##### Tree Growth Module
+
+This module deals with the small trees growth curve ID like black spruce. There is a peatland pre-defined forest growth curve of black spruce. It takes the current eco boundary variable name and then checks if it was changed or just set. As output, it updates the small tree growth curve parameter. It calculates the biomass transfer between various tree pools like stem wood, coarse root, with the help of the age of the small tree.
+
+##### Stand Maturity Module
+
+This module creates the stand growth curve. It gets the standard growth curve ID associated with the pixel, processes it and converts the yield volume to carbon curves.
+
+##### Tree Yield Table Module
+
+A yield table is a table which indicates the volume of wood per unit area of forest that is expected to be at different ages compared to other trees.
+
+This module creates the yield tables for multiple hardwood and softwood species components depending upon the age of the tree, species type and species ID of the tree.
+
+It also gives a summary of total volume of the species growth curve and creates a yield table for a specific type with the transform result yield data.
+
+##### Yield Table Growth Module
+
+This module creates and updates yield tables for different land types like peatland and moss, and transfers content between various biomass pools. It requires the growth yield curve to work which it retrieves from the stand growth curve factory file. It also performs turnover and decay of different pools in its one time step.
+
+The [Turnover module](#turnover-module) and the [Decomposition module](#decomposition-module) are also used as a pre-requisite for the GCBM.
+
+#### Input
+
+There are no specific inputs to the GCBM module. It contains several modules which have multiple inputs and they all interact with each other to produce the output of GCBM in the form of maps and tables.
+
+#### Output
+
+GCBM gives us the estimate of carbon flux generated, Aboveground Biomass etc.
+
+|![Diagram depicting how the GCBM output is displayed on a chart.](../static/img/gcbm-chart-module-case-study.png) |
+|:--:|
+| Diagram depicting how the GCBM output is displayed on a chart. |
+
+|![Diagram depicting how the GCBM output is displayed on a map.](../static/img/gcbm-map-module-case-study.png) |
+|:--:|
+| Diagram depicting how the GCBM output is displayed on a map. |
+
+There is a special repository to visualize the output generated by GCBM [Taswira](https://github.com/moja-global/GCBM.Visualisation_Tool).
```

Comment:
I'd recommend this move to before the discussion of the sub-modules and under the main GCBM module section.